paasta_tools/utils.py

```diff
@@ -465,9 +465,46 @@ class NoSuchLogLevel(Exception):
     pass
 
 
+# The active log writer.
+_log_writer = None
+# The map of name -> LogWriter subclasses, used by configure_log.
+_log_writer_classes = {}
+
+
+def register_log_writer(name):
```

Comment:
This is very similar to register_log_reader in paasta-cli logs, what do you think about a catchall logging.py to hold all the logging related stuff?